### ISSUE TITLE
Open issues from Livestream/scredis

### ISSUE THREAD

**Issue by jastice** [scredis/scredis#6]
Review and reopen unhandled issues from Livestream/scredis if warranted:
- [ ] https://github.com/Livestream/scredis/issues/34
- [ ] https://github.com/Livestream/scredis/issues/40
- [ ] https://github.com/Livestream/scredis/issues/45
- [ ] https://github.com/Livestream/scredis/issues/47
- [x] https://github.com/Livestream/scredis/issues/50
- [x] https://github.com/Livestream/scredis/issues/52
- [x] https://github.com/Livestream/scredis/issues/54

**Comment by kpbochenek** [comment]
Livestream/scredis#47 - Unable to reproduce, issue against Redis 2.x not supported anymore.
Livestream/scredis#45 - Unable to reproduce, 5 years old issue.
Livestream/scredis#40 - Right now doing 10k GETs or 10k SUBSCRIBE in PARALLEL takes the same amount of time(1s). Doing 10k GETs or 10k SUBSCRIBE sequentially takes around 70s in both cases. That's expected behaviour of redis server and scredis. No longer an issue with current scredis/redis.
Livestream/scredis#34 - 5 years old issue, if anyone encounters it again please open a new issue and we can work on that if we can reproduce it.

If anything from a list is encountered in latest version of scredis please open a new issue in this repository.
